contracts/src/v0.8/automation/dev/v2_2/AutomationRegistryLogicB2_2.sol

```diff
@@ -438,7 +442,8 @@ contract AutomationRegistryLogicB2_2 is AutomationRegistryBase2_2 {
       transcoder: s_storage.transcoder,
       registrars: s_registrars.values(),
       upkeepPrivilegeManager: s_storage.upkeepPrivilegeManager,
-      reorgProtectionEnabled: s_hotVars.reorgProtectionEnabled
+      reorgProtectionEnabled: s_hotVars.reorgProtectionEnabled,
+      chainModule: s_hotVars.chainModule
```

Comment:
is getState() called by the plugin? If so, should we try not to change the function signature?

Reply:
will refactor to keep onchain struct compatible between versions.

Reply:
Similar to reorgProtectionEnabled in https://github.com/smartcontractkit/chainlink/pull/11965 , will remove the chainModule in OnchainConfig(Legacy) and add a getChainModule func. cc @RyanRHall @infiloop2

Reply:
perfect @shileiwill